### PR TITLE
合并ocgi/general-pod-autoscaler到bk-bcs/bcs-k8s/bcs-general-pod-autoscaler

### PR DESCRIPTION
**Pull request by Super-zhw**
如题

### CODE REVIEW

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/examples/webhook.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: autoscaling.bcs.dev/v1alpha1
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/examples/webhook.yaml:1]: apiVersion：autoscaling.bkbcs.tencent.com/v1alpha1

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/go.mod [hunk 2]
```diff
@@ -0,0 +1,40 @@
+module github.com/Tencent/bk-bcs/bcs-k8s/bcs-k8s-general-pod-autoscaler
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/go.mod:1]: bcs-k8s-general-pod-autoscaler修正为bcs-general-pod-autoscaler

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/cmd/gpa/validator/server.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The BCS Authors.
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/cmd/gpa/validator/server.go:1]: 开源协议头参照：
https://github.com/Tencent/bk-bcs/blob/master/bcs-services/bcs-cluster-manager/main.go

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/LICENSE [hunk 4]
```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/LICENSE:1]: 文件清理

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/README.md [hunk 5]
```diff
@@ -0,0 +1,684 @@
+## Introduction
+
+General Pod Autoscaler(GPA) is a extension for [K8s HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/), which can be used not only for serving, also for game.
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/README.md:3]: 更换为中文

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/cmd/gpa/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,5 @@
+FROM centos:centos7
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/cmd/gpa/Dockerfile:1]: Dockerfile放置参考https://github.com/Tencent/bk-bcs/blob/master/install/conf/bcs-k8s-master/bcs-gamestatefulset-operator/Dockerfile

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/examples/metric.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+apiVersion: autoscaling.bcs.dev/v1alpha1
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/examples/metric.yaml:1]: 文档目录，example，readme等收归至docs/features/bcs-general-pod-autoscaler下

File: bcs-k8s/bcs-k8s-general-pod-autoscaler/manifeasts/crd.yaml [hunk 8]
```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-general-pod-autoscaler/manifeasts/crd.yaml:1]: manifeasts内容以helm chart方式收录至install/helm/bcs-general-pod-autoscaler下

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=960) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=960) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs?pullRequest=960) it.</sub>

**Review by DeveloperJim — APPROVED** [review]
LGTM

**Pull request merged**